app/views/oauth/authorized_applications/index.html.haml

```diff
@@ -6,7 +6,7 @@
 %hr.spacer/
 
 .applications-list
-  - @applications.each do |application|
+  - @applications.order(created_at: :desc).each do |application|
```

Comment:
This would be better done in the controller.

Reply:
Yes, you are right! Thanks